servers/lambda/src/main/resources/jni-config.json

```diff
@@ -0,0 +1,46 @@
+[
```

Comment:
can you elaborate why this is needed?

Reply:
Not seen my comment above??

Reply:
hm weird, Intellij didn't show me your comments when reviewing the PR